features/meerdere-bsns.feature

```diff
@@ -0,0 +1,122 @@
+#language: nl
+
+Functionaliteit: Als gebruiker wil ik het gezag van meerdere personen in één request kunnen vragen
+
+
+    Achtergrond:
+      Gegeven de persoon 'P1' met burgerservicenummer '000000012'
+      * is meerderjarig
+      * heeft geslachtsnaam gevuld
+      En de persoon 'P2' met burgerservicenummer '000000024'
+      * is meerderjarig
+      * heeft geslachtsnaam gevuld
+      En persoon 'P1' heeft geregistreerd partnerschap met persoon 'P2'
+      | datum huwelijkssluiting/aangaan geregistreerd partnerschap (06.10) |
+      | gisteren - 20 jaar                                                 |
+      En de persoon 'P3' met burgerservicenummer '000000036'
+      * is in Nederland geboren
+      * is minderjarig
+      * heeft persoon 'P1' als ouder1 vanaf de geboorteaangifte
+      * heeft persoon 'P2' als ouder2 vanaf de geboorteaangifte
+      * is ingeschreven in de de BRP
+      En de persoon 'P4' met burgerservicenummer '000000048'
+      * is in Nederland geboren
+      * is meerderjarig
+      * heeft persoon 'P1' als ouder1 vanaf de geboorteaangifte
+      * heeft persoon 'P2' als ouder2 vanaf de geboorteaangifte
+      * is ingeschreven in de de BRP
+      En persoon 'P1' heeft een kind 'P3'
```

Comment:
Wordt met deze stap gezegd dat het ook mogelijk is dat bij een persoon is opgenomen dat hij een ouder1/ouder2 heeft, maar bij de betreffende ouder(s) de persoon niet als kind is opgenomen?

Reply:
ik denk dat het in dit geval beter is om in alle gevallen te redeneren vanuit het kind. Want dat is ook wat de gezagsmodule doet.

Dus redeneren vanuit: En persoon 'Kind' heeft als ouder 'Ouder'
Goed om consistent te zijn denk ik